catalogue/webapp/services/prismic/fetch/events.ts

```diff
@@ -0,0 +1,29 @@
+import { PrismicResultsList, PrismicApiError } from '../types';
+import { Event } from '../types/event';
+import { prismicGraphQLClient, prismicApiError } from '.';
+import { transformPrismicResponse } from '../transformers';
+
+export type PrismicQueryProps = {
+  query: string;
+  pageSize: number;
+  type?: string;
+};
+
+export async function getEvents({
+  query,
+  pageSize,
+}: PrismicQueryProps): Promise<PrismicResultsList<Event> | PrismicApiError> {
+  try {
+    const res = await prismicGraphQLClient('events', query, pageSize);
+    const { allEventss } = await res;
+    const exhibitions = await transformPrismicResponse(allEventss);
```

Comment:
Should this be `events`?

But I'm also wondering if it couldn't all be just one function, we only change the first passed parameter in `prismicGraphQLClient`, right?

Reply:
Yep! I've got some commits to push shortly that should see this working

Reply:
OK I'll hold! 👏 🙌